projects/core/src/window/window-ref.ts

```diff
@@ -58,6 +59,21 @@ export class WindowRef {
     return this.nativeWindow ? this.nativeWindow.localStorage : undefined;
   }
 
+  /**
+   * Returns the window/document location, unless it's not available in the process (i.e. SSR).
+   * When there's no access to the location object, we mimic the location partially, by resolving
+   * the request url (`serverRequestUrl`) and origin (`serverRequestOrigin`) from the injector.
+   *
+   */
+  get location(): Partial<Location> {
+    return !this.isBrowser()
```

Comment:
[medium] Typo. Should be the opposite (document.location should be used in browser)
```suggestion
    return this.isBrowser()
```